crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs

```diff
@@ -269,7 +270,18 @@ impl KitsuneP2pActor {
                             let evt_sender = &evt_sender;
                             match event {
                                 MetaNetEvt::Connected { remote_url, con } => {
-                                    let _ = i_s.new_con(remote_url, con).await;
+                                    if matches!(
+                                        host.is_blocked(con.clone().into(), Timestamp::now()).await,
+                                        Ok(false)
+                                    ) {
+                                        // Con is definitely not blocked.
+                                        let _ = i_s.new_con(remote_url, con).await;
+                                    }
+                                    else {
+                                        // Con is either definitely blocked or
+                                        // we don't know, so drop it.
+                                        let _ = i_s.del_con(remote_url).await;
```

Comment:
As far as I can tell, all del_con does ultimately is mark it closed in the metrics:

https://github.com/holochain/holochain/blob/ffc965d592a8467bd264f7ab3e33bdd46125f5c1/crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs#L597

We should go ahead and call close on the connection, and add a backlog item to make that do something in tx5:

https://github.com/holochain/holochain/blob/ffc965d592a8467bd264f7ab3e33bdd46125f5c1/crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs#L148